<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inscription</class>
 <widget class="QDialog" name="Inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pr√©nom</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mdp1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mdp2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_sexe">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sexe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_login">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_nom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_prenom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mdp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mdp2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Homme</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Femme</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="valider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_datenaiss">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_date_naiss">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cursus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_cursus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cursus</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_branche">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_branche">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Branche</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_filiere">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_filiere">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filiere</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_semestre">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_semestre">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Semestre Actuel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images/Utc.qrc">:/images/fond_inscription.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_login</zorder>
  <zorder>label_nom</zorder>
  <zorder>label_prenom</zorder>
  <zorder>label_mdp1</zorder>
  <zorder>label_mdp2</zorder>
  <zorder>label_sexe</zorder>
  <zorder>input_login</zorder>
  <zorder>input_nom</zorder>
  <zorder>input_prenom</zorder>
  <zorder>input_mdp</zorder>
  <zorder>input_mdp2</zorder>
  <zorder>label_titre</zorder>
  <zorder>comboBox</zorder>
  <zorder>valider</zorder>
  <zorder>label_email</zorder>
  <zorder>input_email</zorder>
  <zorder>label_datenaiss</zorder>
  <zorder>input_date_naiss</zorder>
  <zorder>comboBox_cursus</zorder>
  <zorder>label_cursus</zorder>
  <zorder>comboBox_branche</zorder>
  <zorder>label_branche</zorder>
  <zorder>comboBox_filiere</zorder>
  <zorder>label_filiere</zorder>
  <zorder>comboBox_semestre</zorder>
  <zorder>label_semestre</zorder>
 </widget>
 <resources>
  <include location="../images/Utc.qrc"/>
 </resources>
 <connections/>
</ui>
